lib.php

```diff
@@ -75,15 +75,11 @@ function logstore_xapi_get_cohort_members($cohortids) {
         // Validate params.
         $cohort = $DB->get_record('cohort', array('id' => $cohortid), '*', MUST_EXIST);
         if (!empty($cohort)) {
-            $cohortmembers = $DB->get_records_sql("SELECT u.id, u.email
+            $cohortmembers = $DB->get_records_sql("SELECT u.*
                 FROM {user} u, {cohort_members} cm
                 WHERE u.id = cm.userid AND cm.cohortid = ?
                 ORDER BY lastname ASC, firstname ASC", array($cohort->id));
```

Comment:
Could you reorganise this sql's coding style to follow Moodle docs?
https://docs.moodle.org/dev/SQL_coding_style